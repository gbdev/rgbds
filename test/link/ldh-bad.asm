SECTION "bad", ROM0
	ldh [$1234+@], a
