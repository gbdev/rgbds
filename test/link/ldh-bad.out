error: Address $1234 for `LDH` is not in HRAM range; use `LD` instead
    at ldh-bad.asm(2)
Linking failed with 1 error
